From 9d4a6d9cc3466624613b6c70a49524daf31829db Mon Sep 17 00:00:00 2001
From: chenglulu <chenglulu@loongson.cn>
Date: Fri, 6 Aug 2021 17:43:23 +0800
Subject: [PATCH 10/10] LoongArch: Always existence of indirect function
 support and allow the use of the resolver attribute.

	gcc/
	config.gcc: Indirect function support.
---
 gcc/config.gcc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gcc/config.gcc b/gcc/config.gcc
index e8f8c3cf976..1b1563a7eb6 100644
--- a/gcc/config.gcc
+++ b/gcc/config.gcc
@@ -3636,7 +3636,7 @@ case ${target} in
         ;;
 *-*-linux* | *-*-gnu*)
 	case ${target} in
-	aarch64*-* | arm*-* | i[34567]86-* | powerpc*-* | s390*-* | sparc*-* | x86_64-*)
+	aarch64*-* | arm*-* | i[34567]86-* | powerpc*-* | s390*-* | sparc*-* | x86_64-* | loongarch*-*)
 		default_gnu_indirect_function=yes
 		;;
 	esac
-- 
2.30.1

